src/components/LoadingAccounts.tsx

```diff
@@ -0,0 +1,17 @@
+import * as rb from 'react-bootstrap'
+
+interface LoadingAccountsProps {
+  amount?: number
+}
+
+export function LoadingAccounts({ amount = 1 }: LoadingAccountsProps) {
+  return (
+    <div>
+      {[...new Array(amount)].map((a, index) => (
+        <rb.Placeholder key={index} as="div" animation="glow" style={{ marginBottom: 6 }}>
```

Comment:
```suggestion
        <rb.Placeholder key={index} as="div" animation="wave" style={{ marginBottom: 6 }}>
```

Let's be consistent in the animation we use. I don't have a strong preference over "wave" or "glow" but whatever we pick, let's make it consistent across the app.

Reply:
Good catch, changed.